modules/service/src/main/scala/lucuma/odb/graphql/ObsAttachmentRoutes.scala

```diff
@@ -113,6 +115,7 @@ object ObsAttachmentRoutes {
             s
               .updateAttachment(user, programId, attachmentId, fileName, description, req.body)
               .flatMap(_ => Ok())
+              .guarantee(req.body.compile.drain)
               .recoverWith {
                 case EntityLimiter.EntityTooLarge(_) =>
```

Comment:
I recently fixed a bug in the `EntityLimiter` middleware but it's not released yet. I'm surprised if it hasn't been affecting you?
- https://github.com/http4s/http4s/pull/7264